### PR TITLE
Adding some sweet 'aw yiss' action

### PR DESCRIPTION
**Pull request by trlinkin**
This commit adds a new coffee script to bring the goodness of aw yiss to
the Gutterbot consciousness.

### CODE REVIEW

**Comment by trlinkin** [comment]
Maybe this works? I do not know a whole hell of a lot about hubot internals and am just winging it! Enjoy :)

File: hubot-scripts.json [hunk 1]
```diff
@@ -19,5 +19,6 @@
   "rands-when-you-say.coffee",
   "scotch.coffee",
   "uptime.coffee",
-  "wikipedia.coffee"
+  "wikipedia.coffee",
+  "awwyiss.coffee"
```
Comment (eshamow) [inline hubot-scripts.json]: You actually don't need this line - anything in scripts/ gets added by default.

Can you remove that, rebase and I'll merge?

**Comment by trlinkin** [comment]
@eshamow now?

**Comment by eshamow** [comment]
So there's a bug here - if I type aw yisssssss it works, but if I add additional wwwwwwww's to aw I get an extra image.

**Pull request merged**